Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.yml

```diff
@@ -0,0 +1,398 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: PhishLabs IOC EIR
+  version: -1
+configuration:
+- defaultvalue: https://caseapi.phishlabs.com
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- display: User
+  name: user
+  required: true
+  type: 0
+- display: Password
+  name: password
+  required: true
+  type: 4
+- display: Fetches incidents
+  name: is_fetch
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 1 hours
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: fetch_time
+  required: false
+  type: 0
+description: Get live feeds of IOC data from PhishLabs.
```

Comment:
move another fetch parameter to limit the number of incident created by a single fetch-incident iteration